### DISCUSSION TITLE
Why UCX_MAX_EAGER_RAILS default value is 0?

### DISCUSSION THREAD

**Discussion by huzhijiang** [openucx/ucx#9770] [General]
The default UCX_MAX_RNDV_RAILS=2, but why UCX_MAX_EAGER_RAILS default value is 0?

**Comment by ivankochin** [comment]
Default value of `UCX_MAX_EAGER_RAILS` is 1.

See here: https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_context.c#L208
